Automattic/jetpack
# Jetpack Forms: replace email response template with a basic one

Revert the email response template introduced recently due to customer feedback.

This PR replaces the HTML template with a more basic one that doesn't need to be customised, and is similar to what users are used to already.

### Other information:


## Jetpack product discussion
pcsBup-yo-p2#comment-451

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Apply PR and embed a contact form in a post.
Fill in the contact form.
The site owner should receive the response in a basic email, rather than the new HTML email with images.
